components/collections/codepointtrie_builder/src/wasm.rs

```diff
@@ -32,8 +32,13 @@ where
         .to_owned(),
         format!("{}", std::mem::size_of::<T::ULE>() * 8),
     ];
+
+    let trie_type_int = match builder.trie_type {
```

Comment:
nit: move this down to the function call, it's in the middle of `// Set up the execution environment with a WasiState` here

Reply:
Moved this to function call below.